conftest.py

```diff
@@ -86,8 +86,10 @@ def setup_mender_configured(
             filename,
             key_filename=setup_test_container.key_filename,
         )
+        # Install deb package and missing dependencies
         setup_tester_ssh_connection.sudo(
-            "DEBIAN_FRONTEND=noninteractive dpkg -i %s" % filename
+            "DEBIAN_FRONTEND=noninteractive dpkg -i %s || sudo apt-get -f -y install"
```

Comment:
I might be missing something here, but `apt-get install` what?

Reply:
Not super intuitive, but it is a feature for `apt-get` to install upstream missing dependencies from the last `dpkg -i`.

See https://github.com/mendersoftware/mender-dist-packages/pull/30
See https://askubuntu.com/questions/40011/how-to-let-dpkg-i-install-dependencies-for-me

Reply:
@oleorhagen I pulled you into the review for the black part :)

Reply:
Then ten thumps up :+1: *10